### PR TITLE
chore: updating dependabot to support github-actions, python, TS and docker packages

### PR DESCRIPTION
**Pull request by rajpalc7**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "04:00"
+      timezone: "Canada/Pacific"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major", "version-update:semver-minor"]
```
Comment (WadeBarnes) [inline .github/dependabot.yml]:
```suggestion
        update-types: ["version-update:semver-major", "version-update:semver-patch"]
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  - package-ecosystem: "npm"
+    directory: "/load-agent"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "04:00"
+      timezone: "Canada/Pacific"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major", "version-update:semver-minor"]
```
Comment (WadeBarnes) [inline .github/dependabot.yml]:
```suggestion
        update-types: ["version-update:semver-major", "version-update:semver-patch"]
```

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Just a couple changes please @rajpalc7

File: .github/dependabot.yml (hunk 2 above)
Comment (rajpalc7) [reply .github/dependabot.yml]: Hi @WadeBarnes I have made the changes accordingly.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Looks like the trailing newline at the end of the file was removed.  I believe we want to standardize on including it.

There are VSCode settings you should look into applying to automate some formatting:
- Removing trailing whitespace from lines.
- Adding the trailing new line.

**Comment by rajpalc7** [comment]
Hi @WadeBarnes this is ready now

**Review by WadeBarnes — APPROVED** [review]

**Pull request closed without merging**